framework_tvb/tvb/interfaces/rest/commons/strings.py

```diff
@@ -86,6 +86,7 @@ class RestLink(Enum):
     # DATATYPES
     GET_DATATYPE = "/{" + LinkPlaceholder.DATATYPE_GID.value + "}"
     DATATYPE_OPERATIONS = "/{" + LinkPlaceholder.DATATYPE_GID.value + "}/operations"
+    OPERATION_RESULT = "/{" + LinkPlaceholder.DATATYPE_GID.value + "}/result"
```

Comment:
Wouldn't this be better named `DATATYPE_EXTRA_INFO` or something to contain `extra_information` in the string ?